airbyte-integrations/connectors/source-stripe/source_stripe/schemas/persons.json

```diff
@@ -0,0 +1,362 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Persons Schema",
+  "additionalProperties": true,
+  "type": ["null", "object"],
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
+    },
+    "object": {
+      "type": ["null", "string"]
+    },
+    "account": {
+      "type": ["null", "string"]
+    },
+    "address": {
+      "type": ["null", "object"],
+      "properties": {
+        "city": {
+          "type": ["null", "string"]
+        },
+        "country": {
+          "type": ["null", "string"]
+        },
+        "line1": {
+          "type": ["null", "string"]
+        },
+        "line2": {
+          "type": ["null", "string"]
+        },
+        "postal_code": {
+          "type": ["null", "string"]
+        },
+        "state": {
+          "type": ["null", "string"]
+        }
+      }
+    },
+    "created": {
+      "type": "number"
+    },
+    "dob": {
+      "type": ["null", "object"],
+      "properties": {
+        "day": {
+          "type": ["null", "string"]
```

Comment:
These properties are integers per https://stripe.com/docs/api/persons/object#person_object-dob-day

Reply:
`day`, `month`, and `year` need to be set to integers.